core/src/main/scala/kafka/server/ConfigAdminManager.scala

```diff
@@ -505,7 +505,7 @@ object ConfigAdminManager {
         }
         case OpType.SUBTRACT => {
           if (!listType(alterConfigOp.configEntry.name, configKeys))
-            throw new InvalidRequestException(s"Config value subtract is not allowed for config key: ${alterConfigOp.configEntry.name}")
+            throw new InvalidConfigurationException(s"Config value subtract is not allowed for config key: ${alterConfigOp.configEntry.name}")
```

Comment:
I think the change makes sense. Do we have any tests which cover these case?

Reply:
Yes, it's already covered in `testInvalidIncrementalAlterConfigs`.